cmd/syncthing/gui.go

```diff
@@ -1023,6 +1026,45 @@ func (s *apiService) getSystemLogTxt(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *apiService) getSupportBundle(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think that in this method we can probably avoid creating all the temp files and instead buffer them in memory as byte slices or `bytes.Buffer` objects. The zip creator can use `bytes.Buffer`s to read from.

That also simplifies error handling and avoids issues with not being able to create temp files and whatnot.